### PR TITLE
FIT-415 Update Licensing Shared Library Component to produce CSV file for Java based projects

### PR DESCRIPTION
**Pull request by SergiiRedchuk**
https://osi-cwds.atlassian.net/browse/FIT-415
Updated the library to support JK1 Gradle License Report Plugin.

### CODE REVIEW

File: src/gov/ca/cwds/jenkins/licensing/LicensingSupportType.groovy [hunk 1]
```diff
@@ -3,6 +3,7 @@ package gov.ca.cwds.jenkins.licensing
 enum LicensingSupportType {
   NONE('None'),
   GRADLE_HIERYNOMUS_LICENSE('Gradle Hierynomus License Plugin'),
+  JK1_GRADLE_LICENSE_REPORT('JK1 Gradle License Report Plugin'),
```
Comment (edgibbs) [inline src/gov/ca/cwds/jenkins/licensing/LicensingSupportType.groovy]: So I guess my question is why don't we just get this working for on java library that produces a CSV file and we'll default to that for all the projects?

Comment (SergiiRedchuk) [reply src/gov/ca/cwds/jenkins/licensing/LicensingSupportType.groovy]: Changed it to use only one gradle plugin producing CSV. Removed previous support of the other plugin.

**Comment by edgibbs** [comment]
Changes look fine, I just don't know if you want it approved/merged yet or you're just working on it still?

**Comment by SergiiRedchuk** [comment]
> Changes look fine, I just don't know if you want it approved/merged yet or you're just working on it still?

I'd like them to be merged and have possible other changes related to reusing CwdsCsvReportRenderer in separate PR.

**Comment by edgibbs** [comment]
Alright to merge after cwds-jobs cleanup.

**Pull request merged**